gcp_variant_transforms/transforms/merge_header_definitions.py

```diff
@@ -0,0 +1,156 @@
+# Copyright 2018 Google Inc.  All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Beam combiner function for merging VCF file header definitions."""
+
+from collections import OrderedDict  # pylint: disable=unused-import
+from collections import namedtuple
+from typing import Dict, Union  # pylint: disable=unused-import
+
+import apache_beam as beam
+from gcp_variant_transforms.beam_io.vcf_header_io import VcfHeader   # pylint: disable=unused-import
+from gcp_variant_transforms.libs.vcf_field_conflict_resolver import VcfParserConstants
+
+HeaderFields = namedtuple('HeaderFields', [VcfParserConstants.NUM,
```

Comment:
Thinking about it more: this should actually be named `Definition`.

Reply:
Done.